src/view/popup-comments-view.js

```diff
@@ -0,0 +1,17 @@
+export const createCommensTemplate = () => (
```

Comment:
По хорошему имя функции должно совпадать с именем файла
```suggestion
export const createPopupCommensTemplate = () => (
```